docs/Learning-Environment-Create-New.md

```diff
@@ -411,7 +411,8 @@ with our Agent code.
 2. Change **Decision Interval** from `1` to `10`.
 3. Drag the Target GameObject from the Hierarchy window to the RollerAgent
     Target field.
-4. Modify the Behavior Parameters of the Agent :
+4. Add the BehaviorParameters Script from ML-Agents -> Scripts -> Policy to the RollerAgent.
```

Comment:
Thanks for making the change! One last thing - Can we change this line to "Add the Behavior Parameters script with the Add Component button on the RollerAgent Inspector."? It's easier than going in the Scripts folder and more consistent with the other directions. Thanks again